### PR TITLE
Add EXO token

### PR DESCRIPTION
**Pull request by ExodusKingslayer**
### 为确保 Token Profile 的顺利通过，务必逐一确保以下检查项并补充项目资料：

**.json 及 .png 文件检查项**
- [x] logo 尺寸：120*120 px
- [x] logo 背景：透明
- [x] 代币有至少一笔成功的转账记录
- [x] 此项对于 ERC-20 项目方：合约地址有大小写（做 checksum）
- [x] .json 和 .png 文件名一致（包括大小写）

**项目资料：**

官方公告：https://exodusnetwork.io/

团队背景：EXODUS NETWORK的工作方式就是将无数闲置的智能设备算力集合起来，通过POSTT算法网络对算力进行评估、重组，并提供给需要算力的商业组织，让整个经济模型通过供需平衡的方式组织起来。

项目基本情况：Exodus利用区块链技术独创性得提出了POSTT（Proof of Space Time and Task）共识算法，构建“交叉火力式组网、全智能设备结点”的双层共识部署架构，依托物联网Multi-Hop技术灵活组网，激活海量的分布式在线智能设备。形成一个去中心化共享算力调配、交易市场。Exodus将透过智能合约高效安全地达成一致共识，并按需分配、透明结算、不需要第三方中介的资源价格定制。服务于智能制造、虚拟AR、影视动漫、人工智能、大数据、科学计算等多个高算力需求产业。

媒体报道：
1.火星财经：https://news.huoxing24.com/20200408112328243536.html
2.币乎：https://bihu.com/article/1145297073
3.BUTKAN:https://bitkan.com/zh/ksite/blog/86694
4.艾克斯欧算力网络：https://mp.weixin.qq.com/s/m5KKQTBJsp_Ut6B8pFeExQ
5.网易：https://c.m.163.com/news/a/F3OB2CS005453943.html?spss=wap_refluxdl_2018&spssid=176403cfdfbe1653d3e29cd7d336e2e3&spsw=2&isFromH5Share=article
6.企鹅号：https://page.om.qq.com/page/OnBtOZVMka7IUODublZ4LjOA0?ADTAG=tgi.wx.share.message

收录的交易所：bvnex、okdax、wbf
转账所需 Gas Limit（对 ERC-20 项目方）：（默认为 60000）

---

### Please check the following to ensure a successful verification of Token Profile

**.json and .png document checklists**
- [x] logo dimensions: 120*120 px
- [x] logo background: Transparent
- [x] At least one transaction on the token contract
- [x] For ERC-20 project teams: Upper and lower capitalized alphabets in token address (For checksum purposes)
- [x] Same document name for both .json and .png (Including capitalization)

**Project background:**

Official announcement:https://exodusnetwork.io/
Project team background:EXODUS NETWORK's working method is to gather the computing power of countless idle smart devices, evaluate and reorganize the computing power through the POSTT algorithm network, and provide it to the business organizations that need computing power, so that the entire economic model can be organized in a balanced supply and demand. .

Project basic information:Exodus uses the originality of blockchain technology to propose a POSTT (Proof of Space Time and Task) consensus algorithm to build a two-layer consensus deployment architecture of "cross-fire networking and fully intelligent device nodes", relying on the Internet of Things Multi-Hop The technology is flexible in networking and activates massive distributed online smart devices. Form a decentralized shared computing power allocation and trading market. Exodus will efficiently and safely reach consensus through smart contracts, and allocate resources on demand, transparently settle, and customize resource prices without third-party intermediaries. Serving multiple industries with high computing power such as intelligent manufacturing, virtual AR, film and television animation, artificial intelligence, big data, and scientific computing.

Media publications:
1.火星财经：https://news.huoxing24.com/20200408112328243536.html
2.币乎：https://bihu.com/article/1145297073
3.BUTKAN:https://bitkan.com/zh/ksite/blog/86694
4.艾克斯欧算力网络：https://mp.weixin.qq.com/s/m5KKQTBJsp_Ut6B8pFeExQ
5.网易：https://c.m.163.com/news/a/F3OB2CS005453943.html?spss=wap_refluxdl_2018&spssid=176403cfdfbe1653d3e29cd7d336e2e3&spsw=2&isFromH5Share=article
6.企鹅号：https://page.om.qq.com/page/OnBtOZVMka7IUODublZ4LjOA0?ADTAG=tgi.wx.share.message

Tradeable exchanges:bvnex、okdax、wbf

Recommended Gas Limit (For ERC-20 project teams) for transaction: (Default is 60,000)

### CODE REVIEW

**Comment by sfsy520** [comment]
Pr中提交的文件有多余，如提交多个token，请分开提交。

**Comment by EXOorg** [comment]
@sfsy520 token信息更新是重新pr还是commit更新

**Comment by sfsy520** [comment]
欢迎在社区中推荐用户使用 imToken 2.0 管理资产。

File: erc20/0x8DD70c183CFa4F077B1711442Ae3D54DBA279367.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "initial_price":{
+        "ETH":"0.00023 ETH",
+        "USD":"0.05 USD",
+        "BTC":"0. 0000054 BTC"
+  },
+  "links": {
+    "twitter": "https://twitter.com/Exodusnet",
+    "telegram": "https://t.me/exodusnet",
+    "github": "https://github.com/exodusnet",
+    "facebook": "https://www.facebook.com/exo.dus.7902",
```
Comment (zhen6939) [inline erc20/0x8DD70c183CFa4F077B1711442Ae3D54DBA279367.json:22]: 最后面多了一个 「,」，导致 json 解析失败

**Pull request merged**